libnavui-maneuver/src/test/java/com/mapbox/navigation/ui/maneuver/LaneIconsProcessorTest.kt

```diff
@@ -0,0 +1,1776 @@
+package com.mapbox.navigation.ui.maneuver
+
+import com.mapbox.api.directions.v5.models.ManeuverModifier.LEFT
+import com.mapbox.api.directions.v5.models.ManeuverModifier.RIGHT
+import com.mapbox.api.directions.v5.models.ManeuverModifier.SHARP_LEFT
+import com.mapbox.api.directions.v5.models.ManeuverModifier.SHARP_RIGHT
+import com.mapbox.api.directions.v5.models.ManeuverModifier.SLIGHT_LEFT
+import com.mapbox.api.directions.v5.models.ManeuverModifier.SLIGHT_RIGHT
+import com.mapbox.api.directions.v5.models.ManeuverModifier.STRAIGHT
+import com.mapbox.api.directions.v5.models.ManeuverModifier.UTURN
+import com.mapbox.navigation.ui.maneuver.model.LaneIconResources
+import com.mapbox.navigation.ui.maneuver.model.LaneIndicator
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class LaneIconsProcessorTest {
```

Comment:
There are no tests for the left driving side, could you add some, especially for the critical edge cases?